src/html.rs

```diff
@@ -393,4 +426,94 @@ test some special html-characters as &lt; &gt; and &amp; but also &quot; and &#x
             .is_some());
         assert!(parser.html.find("cid:").is_none());
     }
+
+    #[async_std::test]
+    async fn test_get_html_empty() {
+        let t = TestContext::new().await;
+        let msg_id = MsgId::new_unset();
+        assert!(msg_id.get_html(&t).await.is_none())
+    }
+
+    #[async_std::test]
+    async fn test_html_forwarding() {
+        // alice receives a non-delta html-message
+        let alice = TestContext::new_alice().await;
+        alice.set_config(Config::ShowEmails, Some("2")).await.ok();
+        let chat = alice.chat_with_contact("", "sender@testrun.org").await;
```

Comment:
Maybe `TestContext::chat_with_contact` should be updated to automatically use the title-cased local part of the address as "name"?  Because clearly we mostly don't care about the name (and I had to go look up why the first argument was deemed irrelevant)  /cc @Hocuri